src/Root.tsx

```diff
@@ -0,0 +1,25 @@
+import {
+  Navigate,
+  Route,
+  HashRouter as Router,
+  Routes,
+} from 'react-router-dom';
+import { App } from './App';
+import HomePage from './pages/HomePage';
+import PeoplePage from './pages/PeoplePage';
+import PageNotFound from './pages/PageNotFound';
+
+export const Root = () => (
+  <Router>
+    <Routes>
+      <Route path="/" element={<App />}>
```

Comment:
You can move all paths in enum or constants and use them in all places in your application.

Reply:
Danil thanks for check.